data/kubemacpool/002-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+{{ if .DataExist }}
+data:
+  END_POOL_RANGE: {{ .EndPoolRange }}
```

Comment:
what happens if not provided? shouldn't we fall back to a default value?

Reply:
I want to implement the logic of the default value into the mac pool and not here.